unstructured/partition/email.py

```diff
@@ -182,6 +182,27 @@ def find_embedded_image(
     return Image(text=image_info[:-1]), element
 
 
+def parse_email(
+    filename: Optional[str] = None,
+    file: Optional[IO] = None,
+) -> Tuple[Optional[str], Message]:
+    if filename is not None:
+        with open(filename, "rb") as f:
+            msg = email.message_from_binary_file(f)
+    elif file is not None:
+        with open(file.name, "rb") as f:
+            msg = email.message_from_binary_file(f)
```

Comment:
any reason why not `email.message_from_binary_file(file)` rather than re-opening the `file` object?

Reply:
ah, that makes sense. i think it is ok to just reopen for now, it won't be open for long. thanks for the detailed explanation.